spec/acceptance/callbacks_spec.rb

```diff
@@ -82,8 +82,8 @@ def confirm!
 
       factory :user do
         after(:stub)   { generate(:sequence_3) }
-        after(:build)  { |user| user.first_name = generate(:sequence_1) }
-        after(:create) { |user, evaluator| user.last_name = generate(:sequence_2) }
+        after(:build)  { |user| user.first_number = generate(:sequence_1) }
+        after(:create) { |user, evaluator| user.last_number = generate(:sequence_2) }
```

Comment:
Unused block argument - `evaluator`. If it's necessary, use `_` or `_evaluator` as an argument name to indicate that it won't be used.<br>Line is too long. [85/80]